core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2202,9 +2202,10 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       //Otherwise, use the entity given in the submission.
       submissionRequest.expression match {
         case None =>
-          if (submissionRequest.entityType.getOrElse("") != rootEntityType)
-            DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.BadRequest, s"Method configuration expects an entity of type ${rootEntityType}, but you gave us an entity of type ${submissionRequest.entityType}.")))
-          else {
+          if (submissionRequest.entityType.getOrElse("") != rootEntityType) {
+            val whatYouGaveUs = if (submissionRequest.entityType.isDefined) s"an entity of type ${submissionRequest.entityType.get}" else "no entity"
+            DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.BadRequest, s"Method configuration expects an entity of type $rootEntityType, but you gave us $whatYouGaveUs.")))
```

Comment:
⭐️

Reply:
this failed ui tests (because the error changed to say `Some(type)`) and i can't tell if that's good or bad

Reply:
I was expressing my appreciation of the val name: `you gave us $whatYouGaveUs`